cmd/server/app/webhook_update.go

```diff
@@ -237,7 +237,7 @@ func wireUpProviderManager(cfg *serverconfig.Config, store db.Store) (manager.Pr
 		nil, // ghProviderService not needed here
 	)
 
-	return manager.NewProviderManager(providerStore, githubProviderManager)
+	return manager.NewProviderManager(ctx, providerStore, githubProviderManager)
```

Comment:
do we want to call Close here as well or do we not care since this is a CLI app?